### DISCUSSION TITLE
How to implement volume control?

### DISCUSSION THREAD

**Discussion by Efesdekjian** [pschatzmann/ESP32-A2DP#371] [Q&A] [answered]
Hello! This repository looks very promising. I was looking through the example codes and most of it made sense to me, being a complete beginner to Arduino and ESP32. However, how do you implement the AVRC commands in the Arduino sketch? I can't really figure it out.

**Comment by pschatzmann** [comment, marked as answer]
I think people would usually link avrc command with buttons, so I suggest that you Google how to __use and debounce buttons__ or use __buttons with interrupts__.

But first test your button code with the help of Serial.print() before you move to the next step.

E.g To use the __increase volume button__ you would just do something like a2dp.setVolume(a2dp.getVolume()+const_offset);  where const_offset is a numeric constant that defines how much the volume should increase with each press.

**Reply by Efesdekjian** [reply]
Oh okay! I'll look into it, thank you so much!
